reconfiguration/cmf/concord.cmf

```diff
@@ -109,10 +109,13 @@ Msg AddRemoveWithWedgeCommand 29 {
 
 Msg AddRemoveWithWedgeStatus 30 {
     uint64 sender_id
+    bool bft_flag
```

Comment:
I have doubts about this flag...
We have a bft flag in the request itself (addRemoveWithWedge). 
Having a flag also here means that the operator needs to remember what was the original command, otherwise, what is the meaning of bft in the status where the original request was without bft flag (and vice versa).
What do you thing about returning the bft flag according to the actually stored cmd?